icu4c/APIChangeReport.md

```diff
@@ -5,471 +5,295 @@
  License & terms of use: http://www.unicode.org/copyright.html
 -->
 
-# ICU4C API Comparison: ICU 74 with ICU 75
+# ICU4C API Comparison: ICU 75 with ICU 76
 
 > _Note_ Markdown format of this document is new for ICU 65.
 
-- [Removed from ICU 74](#removed)
-- [Deprecated or Obsoleted in ICU 75](#deprecated)
-- [Changed in  ICU 75](#changed)
-- [Promoted to stable in ICU 75](#promoted)
-- [Added in ICU 75](#added)
-- [Other existing drafts in ICU 75](#other)
+- [Removed from ICU 75](#removed)
+- [Deprecated or Obsoleted in ICU 76](#deprecated)
+- [Changed in  ICU 76](#changed)
+- [Promoted to stable in ICU 76](#promoted)
+- [Added in ICU 76](#added)
+- [Other existing drafts in ICU 76](#other)
 - [Signature Simplifications](#simplifications)
 
 ## Removed
 
-Removed from ICU 74
+Removed from ICU 75
   
-| File | API | ICU 74 | ICU 75 |
+| File | API | ICU 75 | ICU 76 |
 |---|---|---|---|
-| caniter.h | <tt>static</tt> void icu::CanonicalIterator::permute(UnicodeString&amp;, bool, Hashtable*, UErrorCode&amp;) |  Internal | (missing)
-| gregocal.h | UDate icu::GregorianCalendar::defaultCenturyStart() const |  Internal | (missing)
-| gregocal.h | bool icu::GregorianCalendar::haveDefaultCentury() const |  Internal | (missing)
-| gregocal.h | int32_t icu::GregorianCalendar::defaultCenturyStartYear() const |  Internal | (missing)
-| platform.h | <tt>#define</tt> U_HAVE_INTTYPES_H |  Internal | (missing)
-| platform.h | <tt>#define</tt> U_HAVE_STDINT_H |  Internal | (missing)
+| unistr.h | UNISTR_FROM_STRING_EXPLICIT icu::UnicodeString::UnicodeString(const char16_t*) |  StableICU 2.0 | (missing)
+| unistr.h | UNISTR_FROM_STRING_EXPLICIT icu::UnicodeString::UnicodeString(const uint16_t*) |  StableICU 59 | (missing)
+| unistr.h | UNISTR_FROM_STRING_EXPLICIT icu::UnicodeString::UnicodeString(const wchar_t*) |  StableICU 59 | (missing)
+| unistr.h | UnicodeString&amp; icu::UnicodeString::remove(int32_t, int32_t length=(int32_t)) |  StableICU 2.0 | (missing)
+| unistr.h | UnicodeString&amp; icu::UnicodeString::removeBetween(int32_t, int32_t limit=(int32_t)) |  StableICU 2.0 | (missing)
```

Comment:
These two seem like a tool bug, but I don't know if anyone wants to spend time trying to fiddle with the tool.
These show up as removed+added because the default value expression uses a fancy cast now.
@richgillam also just mention on the download page?
(I can add these there later.)

Reply:
> @richgillam also just mention on the download page?

Sure.